web/client/epics/__tests__/map-test.js

```diff
@@ -15,8 +15,10 @@ const {
     zoomToExtent
  } = require('../../actions/map');
 
+const {MAP_INFO_LOAD_START} = require('../../actions/config');
```

Comment:
i think we have added the mapstore alias

do for new code we can use it
const {MAP_INFO_LOAD_START} = require('@mapstore/actions/config');

can you change that ?
can you do it also for other requires?